src/salesforce_uploader.js

```diff
@@ -12,10 +12,17 @@ export function handler (input:?any, context:?any, callback:Function) {
     callback(new NamedError('inputError', 'Input did not contain filename'))
     return
   }
+
+  if (input.preventUpload !== null && input.preventUpload) {
+    callback(new NamedError('preventUploadError', 'Upload to salesforce disabled.'))
```

Comment:
I think it would be better no never see failed executions in PROD unless it actually failed